### DISCUSSION TITLE
showing Version Info in the theme.

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#2380] [Q&A]
We are trying to show the version of the app on the footer of our page.  Our app has many modules and a theme, so we'll have to pick a module version to be the "app version".  I see a ModuleDefinition table with a version column but no ThemeDefinition table.  I know we roll the nuspec version as well when we create new releases.

Q: Is there a simple way to access the ModuleDefinition version data from a theme?
Q: Is there a way to get the nuspec version string of a theme?

**Comment by leigh-pointer** [comment]
Hi @markdav-is 
keep it simple 

```
protected override async Task OnInitializedAsync()
{
       var CurrentVersion = Assembly.GetAssembly(typeof(Index)).GetName().Version.ToString();
}
```

**Comment by sbwalker** [comment]
@markdav-is Modules and Themes are completely independent concepts in Oqtane. As you mentioned, you can create a single project which contains both modules and themes and these will be compiled into a single assembly. If you are looking to version the entire project as a consolidated unit, I would agree with @leigh-pointer that your best approach would be to use the assembly version.
